app/presenters/two_factor_authentication/selection_presenter.rb

```diff
@@ -2,10 +2,11 @@ module TwoFactorAuthentication
   class SelectionPresenter
     include ActionView::Helpers::TranslationHelper
 
-    attr_reader :configuration
+    attr_reader :configuration, :user
 
-    def initialize(configuration = nil)
+    def initialize(configuration = nil, user = nil)
```

Comment:
Maybe consider making these keyword arguments?
Or could we derive user from `configuration`?